src/raster/r.droka/r.droka.py

```diff
@@ -219,6 +220,7 @@ def main():
         y = float(point.split("|")[1])
         # print x,y,z
         # Calcolo cost (sostituire i punti di partenza in start_raster al pusto di punto)
+        # (Cost calculation (replace the starting points in start_raster at the point of the point))
```

Comment:
Can an Italian-speaking contributor approve this translation? I couldn't understand the original sentence using translation tools and the context.

Reply:
My interpretation is that it uses the points in the raster for start_coordinates in the following r.cost command. So, I'd translate as: ```Cost calculation (split and use the starting points in start_raster at the start_coordinates parameter in r.cost call)```. Does that make sense? Note that I removed the extra parenthesis introduced.

Reply:
The extra parentheses were because I left the original Italian comments there and added my suggestion underneath (I don't speak Italian)

Reply:
Ah, didn't realize you planned to leave the original comment too

Reply:
No, you're right, I wanted to replace them :), In fact, I wanted to replace to English for all the file, but I didn't want to replace it with something that lost a meaning